jwst/refpix/reference_pixels.py

```diff
@@ -1758,6 +1842,7 @@ def reference_pixel_correction(input_dataset):
         Corrected dataset
 
     """
+
     input_dataset.do_corrections()
 
-    return input_dataset
+    return
```

Comment:
So, why is the dataset no longer being returned to the calling step?

Reply:
This was in response to a flake8 item: the returned dataset wasn't being used, so it wasn't necessary to return it.  The dataset that is passed as a function argument is altered in place, so there's no need to return a copy of it.